style.css

```diff
@@ -0,0 +1,468 @@
+    font-size: 14px;
+  }
+  
+  .comment-form {
+    display: flex;
+    align-items: flex-start;
+    border: none;
+  }
+  
+  .form {
+    width: 40px;
+    height: 40px;
+    border-radius: 50%;
+    margin-right: 16px;
+    cursor: pointer;
+  }
+  
+  .comment-form input[type=text] {
+    border-top: none;
+    border-left: none;
+    border-right: none;
+    padding-bottom: 4px;
+    border-bottom: 1px solid;
+    border-color: rgb(134, 129, 129);
+    padding-right: 550px;
+    background-color: transparent;
+    margin-top: 10px;
+  }
+  
+  .comm-cancel {
+    display: flex;
+    margin-top: 50px;
+    margin-right: 480px;
+    gap: 10px;
+    cursor: pointer;
+    position: absolute;
+    right: 5px;
+  }
+  
+  .commenter {
+    display: flex;
+    margin-bottom: 16px;
+    margin-left: 65px;
+  }
+  
+  .author {
+    width: 40px;
+    height: 40px;
+    margin-top: 20px;
+    margin-left: 20px;
+    border-radius: 50%;
+    cursor: pointer;
+  }
+  
+  .comment-contents {
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+    margin-left: 5px;
+    font-size: 11px;
+    cursor: pointer;
+  }
+  
+  .comment-contents span {
+    font-size: 9px;
+    font-weight: lighter;
+  }
+  
+  .body-p {
+    display: flex;
+    position: relative;
+    right: 155px;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 20px;
+    font-stretch: 100%;
+    margin-top: 60px;
+  }
+  
+  .com-action {
+    display: flex;
+    align-items: center;
+    margin-top: 4px;
+    margin-left: -8px;
+  }
+  
+  .com-action button {
+    display: flex;
+    align-items: flex-start;
+    position: relative;
+    top: 50px;
+    right: 715px;
+    margin-right: 8px;
+    padding: 0;
+    background: transparent;
+    color: #606077;
+    border: none;
+    cursor: pointer;
+  }
+  
+  .sidebar {
+    width: 50%;
+    padding: 24px 24px 0 0;
+    margin-top: 5px;
+  }
+  
+  .side-bar {
+    margin-top: 8px;
+  }
+  
+  .side-bar:first-child {
+    margin-top: 0;
+    margin-bottom: 8px;
+  }
+  
+  .side-bar a {
+    display: flex;
+    text-decoration: none;
+  }
+  
+  .video-thumb {
+    position: relative;
+    width: 168px;
+    margin-right: 8px;
+  }
+  
+  .overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    transition: 0.3s ease;
+  }
+  
+  .side-bar:hover .overlay {
+    opacity: 0.8;
+  }
+  
+  .overlay svg {
+    position: absolute;
+    top: 4px;
+    right: 4px;
+    width: 16px;
+    height: 16px;
+    padding: 6px;
+    fill: white;
+    background-color: black;
+    border-radius: 3px;
+  }
+  
+  .overlay svg:last-child {
+    top: 34px;
+    right: 4px;
+  }
+  
+  .video-length {
+    position: absolute;
+    bottom: 8px;
+    right: 4px;
+    color: white;
+    padding: 2px 4px;
+    border-radius: 3px;
+    background-color: black;
+    font-size: 12px;
+    font-weight: 500;
+    opacity: 0.9;
+  }
+  
```

Comment:
Please add the correct(straight) video length to your design and try to stick as much as possible.
![159273389-47c601da-ce9a-4e09-8da5-6cbdea56dd98](https://user-images.githubusercontent.com/97598663/185981651-0af10131-200a-4656-ac49-84db70709fd8.jpeg)